backend/test/removeReported.test.js

```diff
@@ -0,0 +1,48 @@
+const removeReported = require('../removeReported');
+
+describe('getting stored reported users', () => {
+
+    console.log("no email")
+    var resultArray = removeReported("")
```

Comment:
Codacy has a fix for the issue: [Replace `··var·resultArray·=·removeReported("")` with `var·resultArray·=·removeReported("");`](https://app.codacy.com/gh/karatwan/CPEN321_world_resources/pullRequest?prid=9811192)
```suggestion
  var resultArray = removeReported("");
```